git_machete/git_operations.py

```diff
@@ -630,7 +640,12 @@ def get_counterpart_branches() -> Iterator[str]:
         all_branches: List[str] = local_branches + counterpart_branches
 
         # The trailing '--' is necessary to avoid ambiguity in case there is a file called just exactly like one of the branches.
-        entries = utils.get_non_empty_lines(self._popen_git("reflog", "show", "--format=%gD\t%H\t%gs", *(all_branches + ["--"])).stdout)
+        entries = utils.get_non_empty_lines(self._popen_git(
```

Comment:
Actually, if you could revert these lines (here and below) &mdash; I'd say that the single-line format is easier to read.